data/network/src/commonMain/kotlin/social/androiddev/common/network/model/Application.kt

```diff
@@ -18,12 +18,14 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class Application(
     // required attributes
+    @SerialName("id") val id: String,
```

Comment:
Neat! Assume this is JSON?

Reply:
Yes, this is from `kotlinx.serialization`. I wonder if we don't need the annotation and by default, it uses reflection on the field name... I hate coupling all the data models with these annotations.. especially if we don't need them.

Reply:
Where is this `id` coming from? It is not found in the documentation: https://docs.joinmastodon.org/entities/application/

The annotation can be omitted if it is reflecting the name.
For `client_id` (snake_case) for example I would add it explicit since it clashes with the kotlin code style (camelCase)

Reply:
@thebino https://docs.joinmastodon.org/methods/apps/#create-an-application

Reply:
Maybe the entity documentation is not up-2-date: https://docs.joinmastodon.org/entities/application/

Reply:
yeah not sure - but I will confirm if `id` is returned from real call and update this model accordingly

Reply:
The `SerialName` annotation is only needed when the variable name is different to the attribute we parse. If we receive "id" I would remove the `SerialName`, but if we receive `client_id` I'd keep it to avoid having snake_case variable names

Reply:
Yes - I will do an isolated PR where I clean up the models and only add annotations to fields that are snake_case.

Reply:
OK, real JSON response shows `id`, so we should note that the documentation is not up-to-date moving forward.

Here's curL you can run if anyone is curious
```
curl -X POST \
	-F 'client_name=Test Application' \
	-F 'redirect_uris=urn:ietf:wg:oauth:2.0:oob' \
	-F 'scopes=read write follow push' \
	-F 'website=https://myapp.example' \
	https://mastodon.example/api/v1/apps
```